CHANGELOG

```diff
@@ -1,3 +1,6 @@
+3.7.0 (2024-07-30):
+    * Drop support of SF 7
```

Comment:
Hello dear mister le V ! I bet you mean drop sf 4 support and add support of sf 7